src/main.cpp

```diff
@@ -933,17 +940,37 @@ bool AcceptToMemoryPoolWorker(CTxMemPool &pool,
     return true;
 }
 
+TransactionClass ParseTransactionClass(const std::string &s)
+{
+    std::string low = boost::algorithm::to_lower_copy(s);
+    if (low == "nonstandard")
+    {
+        return TransactionClass::NONSTANDARD;
+    }
+    if (low == "standard")
+    {
+        return TransactionClass::STANDARD;
+    }
+    if (low == "default")
+    {
+        return TransactionClass::DEFAULT;
+    }
+
+    return TransactionClass::INVALID;
+}
```

Comment:
I didn't see any regression tests for the invalid case

Reply:
thx! Added